README.md

```diff
@@ -6,18 +6,9 @@ It is the first test version developed for the BACKEND-DEVELOPMENT course taught
 
 
 
-<br>
-## Initialize database and migrations
-#### before run the app, please run below command
 
-    flask db init - this will create a folder inside the app directory.
-    
-    flask db migrate - this will create all the migrations.
-    
-    flask db upgrade
-    
 
-<br>
```

Comment:
The README is the place where one goes to and starts
to see how to execute the program and would be perfect not to have
old state or copy pasted information, because the person looking 
at the code keeps trying to follow the README until the program works
but when the README is wrong its hard to start even and people
usually discard the software. No time for that in the sense :)